hack/virtual-cluster-redeploy.sh

```diff
@@ -19,10 +19,14 @@ if [ $CLUSTER_TYPE == "openshift" ]; then
   echo ${auth} > registry-login.conf
 
   internal_registry="image-registry.openshift-image-registry.svc:5000"
-  pass=$( jq .\"$internal_registry\".password registry-login.conf )
+  pass=$( jq .\"image-registry.openshift-image-registry.svc:5000\".auth registry-login.conf  )
```

Comment:
you can use jq to slice the string value e.g ` jq '.[1:-1]'` then pipe to base64. just suggestion current way also works.

Reply:
I will leave this one for now if that is ok :)